<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HostSettingPopup</class>
 <widget class="QWidget" name="HostSettingPopup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>381</width>
     <height>281</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="portLabel">
        <property name="text">
         <string>Port Number</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="port"/>
      </item>
      <item>
       <widget class="QLabel" name="portWarning">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="chatCapacityLabel">
        <property name="text">
         <string>Chat Capacity</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="chatCapacity"/>
      </item>
      <item>
       <widget class="QLabel" name="chatCapacityWarning">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="commandCharLabel">
        <property name="text">
         <string>Command Character to use</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="commandChar"/>
      </item>
      <item>
       <widget class="QLabel" name="commandCharWarning">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="5" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="confirmButton">
        <property name="text">
         <string>Confirm</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cancelButton">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0" rowspan="4">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Orientation::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
